app/fn-guru/src/main/frontend/src/types/namespace.ts

```diff
@@ -51,5 +51,22 @@ export class NamespaceNode {
     }
 }
 
+export class FeatureHotObject extends HotObjectBuilder<number> {
+
+    constructor(features: Features = null) {
+        super(features);
+    }
+
+    setFeature(key: string){
+        super.set(key,0)
+    }
+
+
+    * [Symbol.iterator](): IterableIterator<string> {
+        for (const [k, v] of Object.entries(super.get())) {
+            yield k
+        }
+    }
+}
```

Comment:
Let's not introduce the concept of HotObject to the FrontEnd - and please keep this thing simple

why do we need to do something complicated like
```

    * [Symbol.iterator](): IterableIterator<string> {
        for (const [k, v] of Object.entries(super.get())) {
            yield k
        }
    }

```
?